.env.example

```
@@ -0,0 +1,41 @@
+MYSQL_PASSWORD=<채울 것>
+MYSQL_ROOT_PASSWORD=<채울 것>
+MYSQL_PORT=3306
+
+# 앱이 보는 DB 접속 정보 (컨테이너 네트워크 기준)
+DB_HOST=mysql
+DB_PORT=3306
+DB_NAME=app
+DB_USERNAME=app
+DB_PASSWORD=<채울 것>
+# 관측 전용 계정 — SELECT 만 가능해야 한다
+DB_OBS_USERNAME=app
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_OBS_USERNAME key should go before the DB_PASSWORD key

(UnorderedKey)